Document/0x05j-Testing-Resiliency-Against-Reverse-Engineering.md

```diff
@@ -1325,7 +1325,7 @@ Before we describe the usable identifiers, let's quickly discuss how they can be
 
 #### Static Analysis
 
-In the past, Android developers often relied on the Settings.Secure.ANDROID_ID (SSAID) and MAC addresses. However, the behavior of the SSAID has changed since Android O, and the behavior of MAC addresses [changed with the release of Android N](https://android-developers.googleblog.com/2017/04/changes-to-device-identifiers-in.html "Changes in the Android device identifiers"). In addition, there are new [recommendations for identifiers](https://developer.android.com/training/articles/user-data-ids.html "Developer Android documentation - User data IDs") in Google's SDK documentation. These last recommendations boil down to: either use the `Advertising ID` when it comes to advertising - so that a user can decline - or use the `Instance ID` for device identification. Both are not stable accross device upgrades and device-resets, but `Instance ID` will at least allow to identify the current software installation on a device.
+In the past, Android developers often relied on the `Settings.Secure.ANDROID_ID` (SSAID) and MAC addresses. However, the behavior of the SSAID has changed since Android 8.0 (API level 26), and the behavior of MAC addresses [changed with the release of Android 7.0 (API level 25)](https://android-developers.googleblog.com/2017/04/changes-to-device-identifiers-in.html "Changes in the Android device identifiers"). In addition, there are new [recommendations for identifiers](https://developer.android.com/training/articles/user-data-ids.html "Developer Android documentation - User data IDs") in Google's SDK documentation. These last recommendations boil down to: either use the `Advertising ID` when it comes to advertising - so that a user can decline - or the `Instance ID` for device identification. Both are not stable accross device upgrades and device-resets, but `Instance ID` will at least allow to identify the current software installation on a device. The SSAID should only be used in case to detect fraud.

```

Comment:
Instance ID and Advertising ID are not code variables or similar. Please check the re-phrasing after "in Google's SDK documentation,"

```suggestion
In the past, Android developers often relied on the `Settings.Secure.ANDROID_ID` (SSAID) and MAC addresses. However, the behavior of the SSAID has changed since Android 8.0 (API level 26), and the behavior of MAC addresses [changed with the release of Android 7.0 (API level 25)](https://android-developers.googleblog.com/2017/04/changes-to-device-identifiers-in.html "Changes in the Android device identifiers"). In addition, there are new [recommendations for identifiers](https://developer.android.com/training/articles/user-data-ids.html "Developer Android documentation - User data IDs") in Google's SDK documentation, which boil down to: either use the Advertising ID when it comes to advertising - so that a user can decline - or the Instance ID for device identification. Both are not stable across device upgrades and resets, but the Instance ID will at least allow to identify the current software installation on a device. The SSAID should only be used for fraud detection and for sharing state between apps signed by the same developer signing key (both on Android 8.0 (API level 26) and higher).
```

Reply:
Irestructured it a bit more